hw6/Names.java

```diff
@@ -0,0 +1,109 @@
+import java.io.*;
+import java.util.*;
+import java.awt.*;
+
+public class Names {
+   public static final int numOfDecades   = 14;
+   public static final int startingYear   = 1880;
+   public static final int widthPerDecade = 70;
+   
+   public static void main(String[] args) throws FileNotFoundException {
+      Scanner input = new Scanner(new File("names.txt"));
+      Scanner console = new Scanner(System.in);
+      giveIntro();
+      //form searchCombo with name+space+sex (uppercase), read from user output.
+      System.out.print("name? ");
+      String name1 = console.nextLine();
+      String name = (name1.substring(0, 1).toUpperCase() + name1.substring(1));
+      System.out.print("sex (M or F)? ");
+      String sex = console.nextLine().toUpperCase();
+      String searchCombo = name + " " + sex;
+      //find searchCombo in input file. If there is a matching output, construct a drawing panel.
+      //If there is no matching output, print not found message.
+      String line = find(input, searchCombo);
```

Comment:
You don't really need to actually construct the combo here, you could just pass the 2 variable (name and sex) in as is, and compare them individually.